src/components/WorkerPoolQueueEditForm.vue

```diff
@@ -0,0 +1,111 @@
+<template>
+  <p-form class="worker-pool-queue-edit-form" @submit="submit">
+    <p-content>
+      <p-label label="Name" :message="nameErrorMessage" :state="nameState">
+        <p-text-input v-model="name" :state="nameState" />
+      </p-label>
```

Comment:
this is a problem in most places, `p-label` needs to tie `id` to the control so that clicking the label focuses the control. 

the easiest way to accomplish this is using the #default template which provides a random `id`

```suggestion
      <p-label label="Name" :message="nameErrorMessage" :state="nameState">
        <template #default="{ id }">
          <p-text-input :id="id" v-model="name" :state="nameState" />
        </template>
      </p-label>
```

Reply:
this just utilizes base HTML label for attribute
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#attr-for

Reply:
Valid observation! I went ahead and added it to all fields in all forms I created for worker feature pages